src/main/java/jalau/at18/kingoftokyo/DiceFaceSameCounter.java

```diff
@@ -7,10 +7,10 @@ public DiceFaceSameCounter(DiceHand listDiceHand) { //random list
         this.diceHand = listDiceHand;
     }
 
-    public DiceIndentifier diceFaceCounter() { // grab the random list and count the number of repeated dice faces
-        DiceIndentifier newDiceIdentifier = new DiceIndentifier();
+    public DiceFaceIndentifier diceFaceCounter() {
```

Comment:
Indentifier is spelled wrong, it has to be Identifier

Reply:
ok